packages/ckbtc/candid/minter.d.ts

```diff
@@ -6,16 +6,48 @@ export interface Account {
   subaccount: [] | [Uint8Array];
 }
 export type BitcoinAddress =
+  | { p2wsh_v0: Uint8Array }
```

Comment:
I guess, once merged, we will need the existing PR https://github.com/dfinity/ic-js/pull/370 right?

Reply:
I'm not familiar with those address formats. Why do you think the PRs are related?
And why has that PR been open for so long?

Reply:
> Why do you think the PRs are related?

The PR #370 adds support for p2tr and p2wsh bitcoin adresses validation.
This did change adds p2tr and p2wsh as variant of the bitcoin address in the backend.

> And why has that PR been open for so long?

At that time it wasn't an important PR and @leokazz was suppose to come back to use to give the green light to merge the PR once the ckBTC backend would have been updated on mainnet. I guess he forgot.

Let's merge both.

Reply:
Thanks for clarifying.